CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ Unreleased section should follow [Release Toolkit](https://github.com/newrelic/r
 
 ## Unreleased
 
+### security
+- fixed a security vulnerability by updating golang.org/x/crypto to v0.31.0, golang.org/x/text to v0.21.0
+
```

Comment:
Shouldn't it be just a `chore(deps)`? I think you can just mention it as a dependency update. IMO, Security updates are when you fix something in code of this repo that could have been exploited.

Reply:
when i use `### dependencies` the `changelog.md` validation is failing. I should be using anyone from the below

- **breaking:** A breaking change on a user-facing API. Rendered first and with a warning sign to quickly catch the attention of the reader.
- **security:** Security fixes that remediate potential or existing security vulnerabilities.
- **enhancement:** New features or improvements to existing ones.
- **bugfix:** Fixes to incorrect behavior of the application.

Reply:
Did you try `### dependency`?

Reply:
made the change in this [commit](https://github.com/newrelic/nri-mssql/pull/191/commits/f414e38b9dcd04d60b6215a7524423f878f80712)